aws/ami/files/scylla_install_ami

```diff
@@ -56,68 +53,66 @@ if __name__ == '__main__':
+    run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup', shell=True, check=True)
+    run('/opt/scylladb/scripts/scylla_sysconfig_setup --ami', shell=True, check=True)
+    run('/opt/scylladb/scripts/scylla_bootparam_setup --ami', shell=True, check=True)
     os.remove('{}/.ssh/authorized_keys'.format(homedir))
     os.remove('/var/lib/scylla-housekeeping/housekeeping.uuid')
 
-    os.remove('/etc/udev/rules.d/80-net-name-slot.rules')
     with open('/etc/default/grub') as f:
         grub = f.read()
     grub = re.sub(r'^GRUB_CMDLINE_LINUX="(.+)"$', r'GRUB_CMDLINE_LINUX="\1 net.ifnames=0"', grub, flags=re.MULTILINE)
     with open('/etc/default/grub', 'w') as f:
         f.write(grub)
-    run('grub2-mkconfig -o /boot/grub2/grub.cfg')
+    run('grub-mkconfig -o /boot/grub/grub.cfg', shell=True, check=True)
 
     with open('/etc/cloud/cloud.cfg') as f:
-        cfg = f.read()
-    cfg2 = re.sub('^ssh_deletekeys:   0', 'ssh_deletekeys:   1', cfg, flags=re.MULTILINE)
-    cfg2 = re.sub('^    name: centos', '    name: scyllaadm', cfg2, flags=re.MULTILINE)
+        lines = f.readlines()
+    with io.StringIO() as s:
+        for l in lines:
+            if not re.match(r'^ - mounts\n$', l):
+                s.write(l)
+        cfg = s.getvalue()
+    cfg = re.sub('^preserve_hostname: false', 'preserve_hostname: false\n\nssh_deletekeys: true', cfg, flags=re.MULTILINE)
+    cfg = re.sub('^     name: ubuntu', '     name: scyllaadm', cfg, flags=re.MULTILINE)
     with open('/etc/cloud/cloud.cfg', 'w') as f:
-        f.write(cfg2)
-    shutil.move('{}/99_user_alias.cfg'.format(homedir), '/etc/cloud/cloud.cfg.d')
-
-    # We need to install LTS kernel for the AMI.
-    # However, elrepo-kernel only provides x86_64 package,
-    # and centos-kernel (altarch/7/kernel) only provides aarch64 package.
-    # So we have to use different kernel here.
-    if platform.machine() == 'x86_64':
-        run('yum -y install grubby')
-        run('rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org')
-        run('rpm -Uvh http://www.elrepo.org/elrepo-release-7.0-3.el7.elrepo.noarch.rpm')
-        run('yum -y --enablerepo=elrepo-kernel install kernel-ml')
-        mlkver = get_kver('/boot/vmlinuz-*el7.elrepo.x86_64')
-        run('grubby --grub2 --set-default /boot/vmlinuz-{mlkver}'.format(mlkver=mlkver))
-        with open('/etc/yum.conf', 'a') as f:
-            f.write(u'exclude=kernel kernel-devel')
-    elif platform.machine() == 'aarch64':
-        run('yum -y --enablerepo=centos-kernel update kernel')
+        f.write(cfg)
+    run('cloud-init clean', shell=True, check=True)
+    run('cloud-init init', shell=True, check=True)
+    run('userdel -f ubuntu', shell=True, check=True)
+    run('useradd -o -u 1001 -g scyllaadm -s /bin/bash -d /home/scyllaadm ubuntu', shell=True, check=True)
```

Comment:
where did that 1001 come from?

Reply:
It is 'scyllaadm' UID, because the account added after default user account ('centos' on CentOS, 'ubuntu' on Ubuntu), so it's 1000 + 1.
But I changed the behavor on new version, now the script delete 'ubuntu' or 'centos' account before creating 'scyllaadm' account, so it become UID=1000.